pkg/driver/mount.go

```diff
@@ -39,27 +39,95 @@ const (
 	defaultRegionEnv   = "AWS_DEFAULT_REGION"
 	stsEndpointsEnv    = "AWS_STS_REGIONAL_ENDPOINTS"
 	MountS3PathEnv     = "MOUNT_S3_PATH"
-	hostTokenPathEnv   = "HOST_TOKEN_PATH"
+	hostPluginDirEnv   = "HOST_TOKEN_DIR"
```

Comment:
this is a bit confusing, plugin dir but variable is token dir?

Reply:
Changed variable name to match